google/auth/transport/_mtls_helper.py

```diff
@@ -85,6 +87,90 @@ def _read_dca_metadata_file(metadata_path):
     return metadata
 
 
+def _get_workload_x509(certificate_config_path=None):
+    """Read the cert and key files specified in the certificate config provided.
+    If no config path is provided, check the environment variable and well known
+    gcloud location.
+
+    Args:
+        certificate_config_path (string): The certificate config path. If no path is provided,
+        the well known gcloud location will be used.
+
+    Returns:
+        Tuple[bytes, bytes]: client certificate bytes in PEM format and key
+            bytes in PEM format.
```

Comment:
The function seems to actually return a `Tuple[Optional[bytes], Optional[bytes]]`